tests/EmailValidator/EmailLexerTest.php

```diff
@@ -64,7 +64,7 @@ public function invalidUTF8CharsProvider()
         return $chars;
     }
 
-    protected function utf8Chr($code_point)
+    protected static function utf8Chr($code_point)
```

Comment:
Can you explain the reasoning behind this change please?

Reply:
PhpUnit 10 is deprecating data providers that are not static.
https://github.com/sebastianbergmann/phpunit/commit/9caafe2d49b33a21f87db248a8ad6ca7c7bdac09
https://docs.phpunit.de/en/10.0/writing-tests-for-phpunit.html#data-providers

This will trigger warnings in phpunit testruns. Some CI break with warnings.

Reply:
This particular change is needed to make the data provider static, that is calling this function.